### PR TITLE
add migration

### PR DESCRIPTION
**Pull request by C0Rs1K**

### CODE REVIEW

File: Curtains/Curtains.Infrastructure/InfrastructureConfiguration.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ public static class InfrastructureConfiguration
             services
                 .AddDbContext<CurtainsDbContext>(opt =>
                 {
-                    opt.UseSqlServer(connectionString, opt =>
+                    opt.UseSqlServer("Server=localhost,1433;Database=Curtains;User=sa;Password=<YourStrong@Passw0rd>", opt =>
```
Comment (zd-prog) [inline Curtains/Curtains.Infrastructure/InfrastructureConfiguration.cs]: Why did you remove connection string?

File: Curtains/Curtains.Infrastructure/InfrastructureConfiguration.cs [hunk 2]
```diff
@@ -52,18 +52,7 @@ private static IServiceCollection AddRepositories(this IServiceCollection servic
...
-                            type.Namespace == typeof(ILambrequinsRepository).Namespace || type.Namespace == typeof(IManufacturerRepository).Namespace ||
-                            type.Namespace == typeof(IMarketingInfoRepository).Namespace || type.Namespace == typeof(IMaterialRepository).Namespace ||
-                            type.Namespace == typeof(IOrderRepository).Namespace || type.Namespace == typeof(IOurWorksRepository).Namespace ||
-                            type.Namespace == typeof(IPaymentTypeRepository).Namespace || type.Namespace == typeof(IPillowsRepository).Namespace ||
-                            type.Namespace == typeof(IProductImageRepository).Namespace || type.Namespace == typeof(IProductSetRepository).Namespace ||
-                            type.Namespace == typeof(IReviewRepository).Namespace || type.Namespace == typeof(IUserOrderRepository).Namespace ||
-                            type.Namespace == typeof(IUserRepository).Namespace)
+                            && (type.Namespace == typeof(ICurtainsRepository).Namespace))
```
Comment (zd-prog) [inline Curtains/Curtains.Infrastructure/InfrastructureConfiguration.cs:55]: Does project work without interfaces?

**Review by zd-prog — COMMENTED** [review]
Waiting for answers

File: Curtains/Curtains.Infrastructure/InfrastructureConfiguration.cs (hunk 1 above)
Comment (C0Rs1K) [reply Curtains/Curtains.Infrastructure/InfrastructureConfiguration.cs]: oh my bad :)

File: Curtains/Curtains.Infrastructure/InfrastructureConfiguration.cs (hunk 2 above)
Comment (C0Rs1K) [reply Curtains/Curtains.Infrastructure/InfrastructureConfiguration.cs:55]: yep

Comment (zd-prog) [reply Curtains/Curtains.Infrastructure/InfrastructureConfiguration.cs:55]: Did you check all available endpoints?

**Review by zd-prog — APPROVED** [review]

File: Curtains/Curtains.Infrastructure/InfrastructureConfiguration.cs (hunk 2 above)
Comment (C0Rs1K) [reply Curtains/Curtains.Infrastructure/InfrastructureConfiguration.cs:55]: yeeee, it works :)

**Review by EgorAntonovich — APPROVED** [review]
